### DISCUSSION TITLE
There's always a pause in the video stream when issuing a remote command with Slinger running on Pi 4 to ProHD.

### DISCUSSION THREAD

**Discussion by robosmith** [GerryDazoo/Slinger#495] [General]
@GerryDazoo I am wondering if the Pi 4 is being overloaded temporarily by a remote command, because the stream is always interrupted for a second. I'm guessing that is filling the video buffer of the Player.

Do you have any insight into this phenomena?

**Comment by GerryDazoo** [comment]
It's an artifact of how SlingPlayer and the slingbox handled remote keypresses to minimize latency.  
Nothing I can do about it. It happens on all slingboxes and media players. Your PI is fine.

**Reply by GerryDazoo** [reply]
JIC. What media player are using? Please post the Slinger logs when you use the remote.
